cypress/e2e/data-import/importing-marc-bib-files/single-record-import-with-running-large-job.cy.js

```diff
@@ -71,9 +71,9 @@ describe('ui-data-import: Inventory single record import is not delayed when lar
       Logs.checkFileIsRunning(fileName);
```

Comment:
'getLongDelay' is defined but never used. Allowed unused vars must match /React/u

Reply:
I deleted it